configs/_base_/datasets/imagenet_bs64_convmixer_224.py

```diff
@@ -0,0 +1,71 @@
+_base_ = ['./pipelines/rand_aug.py']
+
```

Comment:
What's the difference with `imagenet_bs64_swin_224.py`? If no difference, just reuse that file.

Reply:
test pipeline resize 233